homework/index.js

```diff
@@ -29,8 +29,42 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoDetails(repo) {
+    const reposList = document.getElementById('reposList');
+    const repoItem = createAndAppend('li', reposList);
+    const commonBlock = createAndAppend('div', repoItem, {
+      class: 'commonBlock',
+    });
+    const dataTypeList = createAndAppend('ul', commonBlock, {
+      class: 'dataTypeList',
+    });
+    createAndAppend('li', dataTypeList, { text: 'Repository:' });
+    createAndAppend('li', dataTypeList, { text: 'Description:' });
+    createAndAppend('li', dataTypeList, { text: 'Forks:' });
+    createAndAppend('li', dataTypeList, { text: 'Updated:' });
+    const dataList = createAndAppend('ul', commonBlock, { class: 'dataList' });
```

Comment:
I couldn't find a `dataList` class in your CSS file.

Reply:
I am not using it for now, but I suggest that this class may be useful in future. But anyway, with a new structure it will be no sense in this class.

Reply:
There is a principle in software development that goes by the acronym YAGNI:

>Yagni originally is an acronym that stands for "You Aren't Gonna Need It". It is a mantra from ExtremeProgramming that's often used generally in agile software teams. It's a statement that some capability we presume our software needs in the future should not be built now because "you aren't gonna need it".

Read more about it here:

https://martinfowler.com/bliki/Yagni.html

Reply:
Okay. Thank you for the explanation!